parts/taxonomy-filters.html

```diff
@@ -0,0 +1 @@
+<!-- wp:wordpress-search/search-taxonomy-filter {"selectedTaxonomies":["document:Category","document:Document Type"]} /-->
```

Comment:
I feel like we don't need this in the parent theme. The Document CPT only exists if the Document Repository plugin is active but is that something that will be common across sites? The `Category` and `Document Type` taxonomies also only exist in FIRB, they're not default taxonomies for the Document CPT.